### PR TITLE
chore: Update wallet, hub-kms and hub-auth dependency

### PR DESCRIPTION
**Pull request by rolsonquadras**
Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

**Review by sudeshrshetty — APPROVED** [review]

File: test/bdd/fixtures/demo/docker-compose-edge-components.yml [hunk 1]
```diff
@@ -292,8 +294,8 @@ services:
       - KMS_SECRETS_DATABASE_TYPE=couchdb
       - KMS_SECRETS_DATABASE_URL=${COUCHDB_USERNAME}:${COUCHDB_PASSWORD}@shared.couchdb:5984
       - KMS_SECRETS_DATABASE_PREFIX=kms
-      - KMS_OPERATIONAL_KMS_STORAGE_TYPE=sds
-      - KMS_OPERATIONAL_KMS_STORAGE_URL=https://edv.trustbloc.local
+      - KMS_KEY_MANAGER_STORAGE_TYPE=sds
```
Comment (fqutishat) [inline test/bdd/fixtures/demo/docker-compose-edge-components.yml:297]: need to switch to EDV

Comment (fqutishat) [reply test/bdd/fixtures/demo/docker-compose-edge-components.yml:297]: @rolsonquadras https://github.com/trustbloc/hub-kms/commit/b10d20af99357fa87a01f1c6219492f9caef139a#diff-523f7c6c3cbacf5edb35be8d2e0a8e1537faa089df69de157f2799137055c442R52

Comment (sudeshrshetty) [reply test/bdd/fixtures/demo/docker-compose-edge-components.yml:297]: I guess @DRK3 is doing that

Comment (rolsonquadras) [reply test/bdd/fixtures/demo/docker-compose-edge-components.yml]: [Done](https://github.com/trustbloc/edge-sandbox/compare/c5715fc798bfde6eacf7ecfd0311ddfc3efaacde..22cdf7fe61f14cc7b1c24938c19bf75a635d23e9)

**Review by fqutishat — APPROVED** [review]

**Pull request merged**